api/sales_dashboard/templates/sales_dashboard/home.html

```diff
@@ -73,14 +73,22 @@ <h1 class="h2">Organisations</h1>
               <td>{{org.id}}</td>
               <td><a href="/sales-dashboard/organisations/{{org.id}}">{{org.name}}</a></td>
               <td>{{ org.subscription.plan|default:"Free" }}{% if org.subscription.payment_method %} - {{ org.subscription.payment_method|capfirst }}{% endif %}</td>
-              <td>{{org.num_users}}</td>
+              <td>
+                {% if org.num_users > org.subscription_information_cache.allowed_seats %}<span class="badge badge-danger">{% endif %}
+                    {{org.num_users}}
+                {% if org.num_users > org.subscription_information_cache.allowed_seats %}</span>{% endif %}
```

Comment:
Does this need to wrap the `{{ org.num_users }}`? Or couldn't we just control the classes themselves via the conditional rather than doing the conditional twice.